docker/jupyter/requirements.txt

```diff
@@ -1,4 +1,3 @@
-shapely
-requests
-numpy
-cassandra-driver==3.9.0
```

Comment:
I'm curious why we were able to remove this? Or why it was needed before but not now?

Reply:
Not sure. The notebook runs just fine without it and it was causing build errors.

Reply:
Possibly the notebook intended to do more. Or the the file was copy pasted from one that did and never checked.